QuoteVerification/QvE/Makefile

```diff
@@ -135,7 +136,8 @@ ENCLAVE_LDFLAGS := -Wl,-z,relro,-z,now,-z,noexecstack -fPIC \
 				   -Wl,-pie,-eenclave_entry -Wl,--export-dynamic -Wl,-Map,qve.map \
 				   -Wl,--defsym,__ImageBase=0  \
 				   -Wl,--build-id		\
-				   -Wl,--version-script=Enclave/linux/qve.lds
+				   -Wl,--version-script=Enclave/linux/qve.lds \
+				   -Wl,--dynamic-linker,
```

Comment:
They are different

 * "-Wl,--dynamic-linker," results in an ELF '.interp' section being present, with zero-length string.
 * "-Wl,--no-dynamic-linker" results in the ELF '.interp' section being entirely absent, and in addition removes the ELF program header

It is possible this difference might not matter, but I don't know enough to be confident suggesting that - similar to how "-Wl,--shared" might another option. It would need an SGX expert to analyse this and make a decision.